projects/core/src/auth/user-auth/services/oauth-lib-wrapper.service.ts

```diff
@@ -29,9 +29,7 @@ export class OAuthLibWrapperService {
     protected authConfigService: AuthConfigService,
     @Inject(PLATFORM_ID) protected platformId: Object,
     protected winRef: WindowRef
-  ) {
-    this.initialize();
```

Comment:
remove this, since initialize() will be called in https://github.com/SAP/spartacus/pull/19144/files#diff-823fabb03957246defb4097e7b4cbd5dd4f996bc0cce847f71ed8ba173d55109R42